src/common/API/storageAPI.tsx

```diff
@@ -0,0 +1,36 @@
+import axios, { AxiosResponse } from "axios";
+import { DocumentStorage, WrappedDocument } from "../../types";
+import { decodeQrCode } from "../utils";
+
+export const getQueueNumber = async (documentStorage: DocumentStorage): Promise<AxiosResponse> => {
+  const url = `${documentStorage.url}/queue`;
+
+  return axios({
+    method: "get",
+    url: url,
+    headers: {
+      "Content-Type": "application/json",
+      "x-api-key": documentStorage.apiKey,
```

Comment:
its possible for an endpoint to not require an api key, not sure what happens here if the value is undefined?

Reply:
hmm, yup i understand, i'll update the code so that only if have api key then it will be in the headers.